cli/src/args/topic.rs

```diff
@@ -112,10 +121,18 @@ pub(crate) struct TopicUpdateArgs {
     pub(crate) topic_id: Identifier,
     /// New name for the topic
     pub(crate) name: String,
+    /// New max topic size
+    /// ("unlimited" or skipping parameter causes removal of max topic size parameter in topic)
```

Comment:
Please add empty line between those lines - after that first line will be printed as a help when `-h` is used and all first line and remaining part will be shown when using `--help`